src/java/org/apache/cassandra/db/compaction/CompactionStrategyManager.java

```diff
@@ -152,10 +154,11 @@ public int getIndexForDescriptor(Descriptor descriptor)
                 return compactionStrategyIndexFor(descriptor);
             }
         };
-        pendingRepairs = new PendingRepairHolder(cfs, router);
+        transientRepairs = new PendingRepairHolder(cfs, router, true);
+        pendingRepairs = new PendingRepairHolder(cfs, router, false);
```

Comment:
fullPendingRepairs and trans[ient]PendingRepairs seemed excessively verbose. fullRepairs and transientRepairs also doesn't work since people usually think of full repairs as non-incremental